app/models/concerns/orchestration/ssh_provision.rb

```diff
@@ -53,7 +53,10 @@ def setSSHWaitForResponse
     end
     self.client = Foreman::Provision::Ssh.new provision_host, image.try(:username), { :template => template_file.path, :uuid => uuid }.merge(credentials)
   rescue => e
-    failure _("Failed to login via SSH to %{name}: %{e}") % { :name => name, :e => e }, e
+    error_message = _("Failed to login via SSH to %{name}: %{e}") % { :name => name, :e => e }
+    self.build_errors = "#{build_errors}\n#{error_message}"
```

Comment:
"Minor nitpick: If build_errors is nil, this interpolation results in a leading newline character. It might be cleaner to use an array join:" 
```suggestion
    self.build_errors = [build_errors, error_message].compact.join("\n")
```